lib/cli/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 const yargs = require('yargs/yargs');
 const path = require('path');
-const Loggerr = require('loggerr');
+const { Loggerr } = require('loggerr');
```

Comment:
gross, why would they make this package less ergonomic in a prerelease

Reply:
I read the source code to fix it

Now `require('loggerr')` return a singleton and the prop return the constructor